sdk/digitaltwins/Azure.ResourceManager.DigitalTwins/api/Azure.ResourceManager.DigitalTwins.netstandard2.0.cs

```diff
@@ -458,9 +459,34 @@ public partial class DigitalTwinsServiceBusProperties : Azure.ResourceManager.Di
         public string PrimaryConnectionString { get { throw null; } set { } }
         public string SecondaryConnectionString { get { throw null; } set { } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct IdentityType : System.IEquatable<Azure.ResourceManager.DigitalTwins.Models.IdentityType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public IdentityType(string value) { throw null; }
+        public static Azure.ResourceManager.DigitalTwins.Models.IdentityType SystemAssigned { get { throw null; } }
+        public static Azure.ResourceManager.DigitalTwins.Models.IdentityType UserAssigned { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.DigitalTwins.Models.IdentityType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.DigitalTwins.Models.IdentityType left, Azure.ResourceManager.DigitalTwins.Models.IdentityType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.DigitalTwins.Models.IdentityType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.DigitalTwins.Models.IdentityType left, Azure.ResourceManager.DigitalTwins.Models.IdentityType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ManagedIdentityReference
```

Comment:
I pretty sure this can be replace by the common [ManagedServiceIdentity](https://apiview.dev/Assemblies/Review/996e07cadc7449809982bc42ec445977#Azure.ResourceManager.Models.ManagedServiceIdentity)

Reply:
It does not have properties holding system assigned identity data. If the `IdentityType` is `SystemAssigned`, how do we get its `PrincipalId` or `TenantId`? @jeffyou543 

Is there a reason that the service does not use the common definition of [ManagedServiceIdentity](https://github.com/Azure/azure-rest-api-specs/blob/878fe925bc9f48b7b1451be3e6842bf44f938c34/specification/common-types/resource-management/v4/managedidentity.json#L49-L75)?

Reply:
The Digital Twins object uses ManagedServiceIdentity. For our code application, the resources that are using the ManagedIdentityReference is a child of the Digital Twin object that is created. We do not need principalId or tenantId in the model for this, since it is a custom model used for our needs in the backend.

Reply:
Since it's different from the common [ManagedServiceIdentity](https://github.com/Azure/azure-rest-api-specs/blob/878fe925bc9f48b7b1451be3e6842bf44f938c34/specification/common-types/resource-management/v4/managedidentity.json#L49-L75), please use a more descriptive name like `DigitalTwinsManagedIdentityReference` and also rename `IdentityType` to something like `DigitalTwinsManagedIdentityType ` through the `rename-mapping` section in `autorest.md`.

Reply:
Updated to use these names